### ISSUE TITLE
CORS Error on Safari web browser

### ISSUE THREAD

**Issue by hugoromo** [odoo/odoo#64975]
Impacted versions:
13
Steps to reproduce:
Type into any input box using Safari browser
Current behavior:
 "Unknown CORS Error" message when trying to type email or anything into any input. And this appears on the Javascript Console: 
`TypeError: undefined is not an object (evaluating 'settings.keyValue.charCodeAt')`
This even happens on the official forum of Odoo
Expected behavior:
Not show any error message
Video/Screenshot link (optional):

Support ticket number submitted via odoo.com/help (optional):

**Comment by pedrobaeza** [comment]
Hi, thanks for reporting. Are you able to reproduce the problem in runbot infrastructure (http://runbot.odoo.com/runbot - click on the blue door icon of the version row - admin/admin)?

If so, please explain with all the possible detailed steps to get to the error. Not having them, it's very difficult to hunt down the problem. Please don't be shy of adding details, and even better if you provide some screenshots or videos.

If not, this can be due to a configuration change, a custom/community module you have deployed in your DB or an outdated code. Please report back if it's the case.

For keeping a clean house in this issue tracker, if you don't answer in several days, this will be closed assuming it was one of the stated things.

**Comment by hugoromo** [comment]
Yes, I get the same error message when trying to type the username "admin" on runbot. I've upload a video. Just typing down anything on safari will throw this error

https://user-images.githubusercontent.com/28797870/106013821-73380c00-6082-11eb-8b24-4f1d0c6f6563.mov
